### DISCUSSION TITLE
File-Download-Example

### DISCUSSION THREAD

**Discussion by mc0110** [miguelgrinberg/microdot#113] [Q&A]
Hi Miguel,

I know this is not an issue. Can you give me a code example for a download. In your examples I find an upload, but no download. Sorry if the question is rather banal.

Best regards
Magnus

**Comment by miguelgrinberg** [comment]
The documentation has a section on [serving files](https://microdot.readthedocs.io/en/latest/intro.html#file-responses). The example there uses an HTML file. You can replace that with any kind of file you want the client to download.

**Comment by mc0110** [comment]
Thank you for this super quick reply and sorry again for the stupid question. I did not see it.

**Comment by miguelgrinberg** [comment]
There are no stupid questions!

**Comment by mc0110** [comment]
Great tip. It works now!!! Thank you very much!
